sk-python-hello-world/.vscode/tasks.json

```diff
@@ -0,0 +1,18 @@
+// Create a task to run poetry install
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "install",
```

Comment:
Confirmed that this command runs. I deleted my `.venv` and `poetry.lock` and pressed F5.  First `poetry install` ran, then it ran `main.py`.